homeworks/vladislav.buzunar_buzyaka/1-basics/basics.js

```diff
@@ -1,31 +0,0 @@
-// Variable numbers
```

Comment:
You should not delete this file. Instead, please, revert the changes for this specific file.